AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Details.cshtml

```diff
@@ -11,7 +13,31 @@
 
 <div class="row">
     <div id="col-main">
-        <div class="col-md-9"><h2>@Model.Name <a asp-controller="Campaign" asp-action="Edit" asp-route-id="@Model.Id" class="btn btn-default" title="Edit"><i class="fa fa-edit"></i></a> <a asp-controller="Campaign" asp-action="Delete" asp-route-area="Admin" asp-route-id="@Model.Id" class="btn btn-default"><i class="glyphicon glyphicon-trash"></i></a></h2></div>
+        <div class="col-md-9">
+            <h2 class="inline-form">
+                @Model.Name
+            </h2>
+            <a asp-controller="Campaign" asp-action="Edit" asp-route-id="@Model.Id" class="btn btn-default inline-form" title="Edit"><i class="fa fa-edit"></i></a>
+            <a asp-controller="Campaign" asp-action="Delete" asp-route-area="Admin" asp-route-id="@Model.Id" class="btn btn-default inline-form"><i class="glyphicon glyphicon-trash"></i></a>
+            @if (User.IsUserType(UserType.SiteAdmin))
+            {
+                <form asp-controller="Campaign" asp-route-area="Admin" asp-action="LockUnlock" class="inline-form">

```

Comment:
Why does this have to be a form? Seems like a lot of extra code. You are already validating that the logged-in user is a SiteAdmin.